src/test/kotlin/coverosR3z/timerecording/TimeRecordingTests.kt

```diff
@@ -75,7 +77,7 @@ class TimeRecordingTests {
         val utils = TimeRecordingUtilities(fakeTimeEntryPersistence)
         val entry = createTimeEntryPreDatabase(time= Time(60 * 2), project= Project(1, "an invalid project"))
 
-        assertThrows(ExceededDailyHoursAmountException::class.java) { utils.recordTime(entry) }
+        assertThrows(AssertionError::class.java) { utils.recordTime(entry) }
```

Comment:
Debatable - the name of the test method at the very least suggests further thought is warranted.

Reply:
I swear I changed this one back after I mocked cua appropriately, surprised it was passing

Reply:
It _is_ an assertion error thrown here

Reply:
Fixed using fakecua